bedrock/firefox/templates/firefox/new/horizon/scene1.html

```diff
@@ -0,0 +1,120 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new" %}
+
+{% extends "firefox/new/horizon/base.html" %}
+
+{# all CSS must be in extrahead block for old IE #}
+{% block extrahead %}
+  {% stylesheet 'firefox_new_horizon_common' %}
+  {% stylesheet 'firefox_new_horizon_scene1' %}
+{% endblock %}
+
+{% block body_id %}firefox-new-scene1{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="main-container">
+    <div class="horizon">
+      <div class="inner-container">
+        <header id="masthead">
+          <div id="tabzilla">
+            <a href="{{ url('mozorg.home') }}" data-link-type="nav" data-link-name="tabzilla">Mozilla</a>
+          </div>
+          <h2>{{ high_res_img('firefox/template/header-logo-inverse.png', {'alt': 'Firefox', 'width': '185', 'height': '70'}) }}</h2>
+          <div class="version-message-container">
+            <div id="version-message-android-latest" class="version-message">
```

Comment:
Semantic nitpick: maybe these should be headings or paragraphs instead of generic divs. I think this is carried over from the older page (pre-rocket, even) but they probably should have been more semantically valuable elements there as well.